manageiq-operator/api/v1alpha1/helpers/miq-components/kafka.go

```diff
@@ -228,7 +240,51 @@ func KafkaDeployment(cr *miqv1alpha1.ManageIQ, scheme *runtime.Scheme) (*appsv1.
 		},
 		Env: []corev1.EnvVar{
 			corev1.EnvVar{
-				Name: "KAFKA_BROKER_USER",
+				Name:  "KAFKA_ENABLE_KRAFT",
```

Comment:
Please move all of the ENV vars into the reconcile function.  You can use `addOrUpdateEnvVar` like [here](https://github.com/ManageIQ/manageiq-pods/blob/93463bb82a05ebb240976514fe361d4add0014b1/manageiq-operator/api/v1alpha1/helpers/miq-components/orchestrator.go#L168) that way if any of these change or become configurable in the future, they will be updated in the deployment.